README.md

```diff
@@ -414,10 +414,12 @@ were necessary.
 Each commit should ideally contain changes that are functionally
 connected/related.  For example, changes to the command line parsing
 code, changes to the documentation, and changes to some unrelated
-comments may be split into three commits.
+comments may be split into three commits. Also note that a single logical
+change (suitable for a single commit) may comprise changes in several
+files.
 
 Learn how to select chunks of changed files to do multiple
-separate commits of unrelated things. This is done using `git commit -p
+separate commits of unrelated things. This can be done using `git add -p
 ...`.
 
 Avoid "force push" unless it makes everyone's life easier.
```

Comment:
```suggestion
Avoid "force push" unless it makes everyone's life easier. And when you do, use `--force-with-lease` to avoid pushing when new changes has happened on the remote.
```

Reply:
Sounds good! Should @pontus merge this?

Reply:
Perhaps add:

> A situation where it makes everyone's life easier is when you're tidying up your own topic branch.

Reply:
@danr Is "Topic branch" what's also called "feature branch"?

Reply:
@danr can you add your suggestions to this PR!?

Reply:
@kusalananda I suppose it is either a feature branch or hotfix branch but I changed it to just branch to not make any confusion.
@jhagberg done